workspaces/arborist/lib/arborist/reify.js

```diff
@@ -123,10 +124,19 @@ module.exports = cls => class Reifier extends cls {
       // we do NOT want to set ownership on this folder, especially
       // recursively, because it can have other side effects to do that
       // in a project directory.  We just want to make it if it's missing.
-      await mkdir(resolve(this.path), { recursive: true })
+      const resolvedPath = resolve(this.path)
+      try {
+        await access(resolvedPath)
+      } catch (accessError) {
+        if (accessError.code === 'ENOENT') {
+          await mkdir(resolvedPath, { recursive: true })
```

Comment:
Just a note for future devs looking at this change.  Any error other than `ENOENT` that `access` gets is going to also be thrown if attempting `mkdir`.  I think only ignoring that error is the correct one.